megabus/src/main/java/com/bazaarvoice/megabus/MegabusBootWorkflowManager.java

```diff
@@ -78,6 +86,7 @@ public void start() throws Exception {
             if (bootStatus == MegabusBootDAO.BootStatus.COMPLETE) {
                 _missingRefDelayService.startAsync();
                 _refResolverService.startAsync();
+                _tableEventProcessorService.startAsync();
```

Comment:
line 116 doesnt exist, but I definitely do need to add a stop for graceful shutdown. That being said, we definitely don't want to remove the table event registration each time we shutdown, as this could lead to missing events.

On the topic of reboot, we will need to add a step to our reboot runbook to remove any existing table events if we are booting if the same applicationID. If we are booting with a different applicationID, there is no need to worry because they will register under different names, and the old applicationId with expire after 7 days.

Reply:
looks like you addressed this in the last commit. 👍